ci/linkcheck.sh

```diff
@@ -3,22 +3,18 @@
 set -e
 set -o pipefail
 
-# https://docs.travis-ci.com/user/environment-variables/#default-environment-variables
-if [ "$TRAVIS_EVENT_TYPE" = "cron" ] ; then # running in cron job
+# https://docs.github.com/en/actions/reference/environment-variables
+if [ "$GITHUB_EVENT_NAME" = "schedule" ] ; then # running in scheduled job
   FLAGS=""
 
   echo "Doing full link check."
   set -x
 elif [ "$CI" = "true" ] ; then # running in PR CI build
-  if [ -z "$TRAVIS_COMMIT_RANGE" ]; then
-    echo "error: unexpected state: TRAVIS_COMMIT_RANGE must be non-empty in CI"
-    exit 1
-  fi
 
-  CHANGED_FILES=$(git diff --name-only $TRAVIS_COMMIT_RANGE | tr '\n' ' ')
+  CHANGED_FILES=$(git diff --name-only origin/master... | tr '\n' ' ')
```

Comment:
Are you sure this is correct? I think `origin` may not always be up to date in the pipeline. You can use `github.event.pull_request.base.sha` instead: https://stackoverflow.com/a/61861763/7669110

Reply:
I think it works with `git fetch origin master` but your suggestion seems better!